pkg/wsman/client/wsman.go

```diff
@@ -127,6 +127,13 @@ func NewWsman(cp Parameters) *Target {
 				config = res.tlsConfig
 			} else {
 				config = &tls.Config{InsecureSkipVerify: cp.SelfSignedAllowed}
+				if cp.AllowInsecureCipherSuites {
```

Comment:
Could you please update the TLS version to VersionTLS12 and use a more specific list of insecure ciphers instead of tls.InsecureCipherSuites()?

I've tested these changes on AMT v12.0.92.2145 and v11.8.86, and they work as expected. Here's the updated code for reference:

```
	if cp.AllowInsecureCipherSuites {
		config.MinVersion = tls.VersionTLS12

		for _, suite := range tls.CipherSuites() {
			config.CipherSuites = append(config.CipherSuites, suite.ID)
		}

		// add specific insecure ciphers
		insecureCiphers := []uint16{
			tls.TLS_RSA_WITH_AES_128_GCM_SHA256,
			tls.TLS_RSA_WITH_AES_128_CBC_SHA,
			tls.TLS_RSA_WITH_AES_256_CBC_SHA,
		}
		config.CipherSuites = append(config.CipherSuites, insecureCiphers...)
	}
```

Reply:
it didn't work with insecure ciphers? Those three are listed in the list: https://cs.opensource.google/go/go/+/refs/tags/go1.24.1:src/crypto/tls/cipher_suites.go;l=80

Reply:
It does work, but I noticed that legacy AMT versions only support these specific ciphers. The trade-off is between limiting insecure ciphers for compatibility versus your approach of adding all insecure ciphers, which reduces maintenance if a cipher is marked as insecure in the future.

I'm okay with adding all the insecure ciphers. Let me know your thoughts!

Reply:
I've updated this with your 3, but yes that is my thinking is that I don't want to have to maintain what is considered insecure.